plugin/cmd/kubectl/utils/utils.go

```diff
@@ -34,63 +34,85 @@ type metadata struct {
 	Name string `json:"name"`
 }
 
+// KubeGet makes a --raw GET call to the specified core api server endpoint.
+// The "endpoint" parameter must begin without a slash and contain both a
+// <resource_kind_plural>/<resource_name>: namespaces/foo
+func KubeGet(endpoint string) (stdout, stderr io.ReadCloser, err error) {
+	v, logLevel := Loglevel()
+	caller := os.Getenv("KUBECTL_PLUGINS_CALLER")
+	kubeCmd := exec.Command(caller, "--namespace", Namespace(), v, logLevel, "get", "--raw", "/api/v1/"+endpoint)
```

Comment:
@fabianofranz A shortcoming with this patch that I have recently noticed, is that for cases like this one, where we use `get --raw` to see if a namespace exists, the success of the command is entirely dependent on whether or not the currently logged in user in kubeconfig has permission to list all namespaces in the cluster